app/views/homeworks/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Homeworks</h1>
```

Comment:
not plural :)

perhaps "Homework Assignments" 

will bust the test cases below

Reply:
Agreed.

Reply:
Changed :)